### PR TITLE
chore(docs): initializes documentation directory

### PR DESCRIPTION
**Pull request by PhilippeMts**
## Description

We here setup a _Doks_ project inside `./docs/`.

CI/CD is configured to deploy it automatically using GH branches.

## Change checklist

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: docs/functions/hi-from-lambda.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+exports.handler = (event, context, callback) => {
```
Comment (jxs) [inline docs/functions/hi-from-lambda.js:1]: is this function required?

File: .github/workflows/gh-pages.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+name: GitHub Pages
+
+on:
+  push:
+    branches:
+      - main  # Set a branch to deploy
+  pull_request:
```
Comment (jxs) [inline .github/workflows/gh-pages.yml:7]: Do we want to deploy the docs on PR's to `main`?

File: docs/netlify.toml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+[build]
```
Comment (jxs) [inline docs/netlify.toml:1]: do we need this file since we are going to be using github pages?

File: docs/package.json [hunk 4]
```diff
@@ -0,0 +1,71 @@
+{
+  "name": "doks-child-theme",
```
Comment (jxs) [inline docs/package.json:2]: this should reflect `kythera` details right?

**Review by jxs — COMMENTED** [review]
Thanks for starting this Philippe, left some notes

File: docs/CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
+### Changelog
```
[the remark this answers is not in the document]
Comment (PhilippeMts) [reply docs/CHANGELOG.md:1]: Thanks for this remark. What is happening actually is:

*Doks* installation instructions advise on cloning their GH repository.

I'm not super comfortable with the idea of then having a repository dedicated to the documentation. It is totally feasible of course, and with git submodules, the flow could be almost seamless. But it stills add friction, does require a dedicated repository, and in the end, I would rather clone the latest version here (thus the readme, changelog, etc.) and remove its history from our repo.

Do you feel like we stopped halfway? Like, either we should keep the history, using submodule, or go the full way, remove the history and all the files referring to Doks history?

To me, the current situation is 'acceptable' but I would really appreciate your recommendation.

File: .github/workflows/gh-pages.yml (hunk 2 above)
Comment (PhilippeMts) [reply .github/workflows/gh-pages.yml:7]: We don't. But I think the `${{ github.ref == 'refs/heads/main' }}` is not met on such PR's, thus preventing the actual deployment.

File: docs/CHANGELOG.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (jxs) [reply docs/CHANGELOG.md:1]: thanks for the explanation Philippe! Yeah agree with you to go for the second option, let's have it here without its history and have only the required files for our doks wdyt?

[the remark this answers is not in the document]
Comment (PhilippeMts) [reply docs/CHANGELOG.md:1]: Great. Doing that, based on the rest of your review 👍

**Review by jxs — APPROVED** [review]
LGTM

**Pull request merged**